packages/trader/src/Modules/Contract/Containers/replay-chart.tsx

```diff
@@ -0,0 +1,179 @@
+                margin={margin || null}
+                isMobile={isMobile()}
+                enabledNavigationWidget={isDesktop()}
+                enabledChartFooter={false}
+                granularity={granularity}
+                requestAPI={wsSendRequest}
+                requestForget={wsForget}
+                requestForgetStream={wsForgetStream}
+                crosshair={isMobile() ? 0 : undefined}
+                maxTick={isMobile() ? 8 : undefined}
+                requestSubscribe={wsSubscribe}
+                settings={settings}
+                startEpoch={start_epoch}
+                scrollToEpoch={scroll_to_epoch}
+                stateChangeListener={chartStateChange}
+                symbol={symbol}
+                allTicks={all_ticks}
+                topWidgets={ChartTopWidgets}
+                isConnectionOpened={is_socket_opened}
+                isStaticChart={
+                    // forcing chart reload when start_epoch changes to an earlier epoch for ACCU closed contract:
+                    is_accumulator_contract && end_epoch && Number(start_epoch) < Number(prev_start_epoch)
+                }
+                shouldFetchTradingTimes={false}
+                should_zoom_out_on_yaxis={is_accumulator_contract}
+                yAxisMargin={getChartYAxisMargin()}
+                anchorChartToLeft={isMobile()}
+                shouldFetchTickHistory={
+                    getDurationUnitText(getDurationPeriod(contract_info)) !== 'seconds' ||
+                    contract_info.status === 'open'
+                }
+                shouldDrawTicksFromContractInfo={is_accumulator_contract}
+                contractInfo={contract_info}
+                contracts_array={getContractsArray()}
+                isLive={!has_ended}
+                startWithDataFitMode={true}
+            >
+                {is_beta_chart &&
+                    markers_array.map(({ content_config, marker_config, react_key }) => (
+                        <ChartMarkerBeta
+                            key={react_key}
+                            marker_config={marker_config}
+                            marker_content_props={content_config}
+                            is_bottom_widget_visible={isBottomWidgetVisible()}
+                        />
+                    ))}
+                {!is_beta_chart &&
+                    markers_array.map(({ content_config, marker_config, react_key }) => (
+                        <ChartMarker
+                            key={react_key}
+                            marker_config={marker_config}
+                            marker_content_props={content_config}
+                            is_bottom_widget_visible={isBottomWidgetVisible()}
+                        />
+                    ))}
+                {!is_beta_chart && is_accumulator_contract && !!markers_array && (
+                    // @ts-expect-error DelayedAccuBarriersMarker doesn't accept undefined in marker_component
```

Comment:
The problem is that after releasing modificated charts, `accu_barriers_marker_component` might be `undefined` (line 255) and for  `<DelayedAccuBarriersMarker />` it is NOT optional. Currently not know how to fix it better, so add a temporary `@ts-expect-error.`
Please, share your thoughts.

Reply:
maybe you want to see if u can change type of `marker_component` prop in `<DelayedAccuBarriersMarker />`

Reply:
Check other locations where the same component is used if used in multiple places

Reply:
The problem is that `marker_component` is used as a returned component
```
const DelayedAccuBarriersMarker = React.memo(
    ({ marker_component: MarkerComponent, ...props }: TDelayedAccuBarriersMarkerProps) => (
        <MarkerComponent {...{ ...props, previous_spot_time: undefined }} />
    ), // barrier range will get updated only when previous_spot_time changes:
    (prevProps, nextProps) => prevProps.previous_spot_time === nextProps.previous_spot_time
);
```

Reply:
I will try not render <`MarkerComponent /> `(return `null`) if `marker_component` is `undefined`, but need to check markers in Contract Details Page for Accumulators. 
Will check tomorrow

Reply:
I've changed type, make it optional, but know we need to make sure, that for countries which supply beta-chart, accumulator markers are working normaly